typedoc.json

```diff
@@ -0,0 +1,21 @@
+{
+  "mode": "modules",
+  "out": "./docs/autogen",
```

Comment:
do you have a preference name for the json file? I call it `output.json` though `api.json` might be more fitting.